docs/en/developer/create-web-component/VUE.md

```diff
@@ -341,61 +396,109 @@ To add Chrome (the standard border around portlets):
 
-```xml
+``` xml
     <parameter>
         <name>chromeStyle</name>
         <value>no-chrome</value>
     </parameter>
 ```
 
+To grant permission to everyone to browse for the web component and select it:
+
+``` xml
+    <group>Everyone</group>
+    <permissions>
+        <permission system="UP_PORTLET_SUBSCRIBE" activity="BROWSE">
+            <group>Everyone</group>
+        </permission>
+    </permissions>
+
+```
+
 ### Add webjar to resource server
 
-In the `overlays/resource-server/build.gradle` file in the uPortal-start
+In the **overlays/resource-server/build.gradle** file in the uPortal-start
 project, add the following runtime dependency:
 
-```gradle
+```
     runtime "org.webjars.npm:uportal__{component-name}:{version}@jar"
 ```
 
 For example:
 
-```gradle
+```
     runtime "org.webjars.npm:uportal__weather-thingy:0.1.0-SNAPSHOT@jar"
 ```
 
-## Appendix
+### Rebuild uPortal-start
 
-### Node.js installation
+Rebuild the **uPortal-start** project to populate the database with the new
+portlet definition and load the new webjar into the resource server.
 
-#### Mac OS X
+```
+./gradlew portalInit
+```
 
-1. [MacPorts](https://www.macports.org/)
-2. [HomeBrew](https://brew.sh/)
-3. macOS installer (.pkg) from [Node.js website](https://nodejs.org/)
+When you start uPortal, you should be able to find the new component when
+you select *Customize > Add Stuff*.
 
-##### With MacPorts
+# Appendix
+
+## A. Quick rebuild and deploy
+
+Once you have built the web component and included it in uPortal, here is
+a way to speed up the rebuild and redeployment process so you can see your
+changes quickly in uPortal.
+
+### Rebuild web component
+
+1\. Re-build the web component.
 
-```bash
-sudo port list | grep node
-sudo port install nodejs10
+```
+npm run build
 ```
 
-##### With Homebrew
+2\. Re-package the WebJar and copy to local Maven repo for uPortal-start to use.
 
-```bash
-brew search node
-brew install node
+```
+./gradlew install
+```
+
+### Redeploy to uPortal resource-server
+
+You can do this *while uPortal is still running* which is why it's much
+faster. You don't have to stop and restart uPortal which can take several
+minutes. This assumes you used the uPortal-start project to run uPortal
+locally with `./gradlew portalInit` and `./gradlew portalOpen` (or
+`./gradlew tomcatStart`) and that *you have not stopped Tomcat* (with
+`./gradlew tomcatStop`).
+
+In the **uPortal-start project** root directory:
+
+3\. Re-build the resource-server so it pulls the new WebJar into its resources.
+
+```
+./gradlew :overlays:resource-server:build
+```
+
+4\. Copy the newly-built **resource-server.war** file into the **tomcat/webapps**
+directory where it will automatically be deployed and overwrite the current one.
+
+```
+cp overlays/resource-server/build/libs/resource-server.war .gradle/tomcat/webapps/
 ```
 
-### build.gradle files
+5\. Refresh the browser and you should see the changes.
+
+## B. Sample build.gradle files
```

Comment:
and then 
```
###  B. Sample build.gradle files
```
🎉 😃 still a good catch to change from number to letters in the appendix, this will be very helpful.

Reply:
Done.